pyan/analyzer.py

```diff
@@ -378,6 +378,16 @@ def visit_ImportFrom(self, node):
         #
         # https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time
         from_node = self.get_node_of_current_namespace()
+        # resolve relative imports 'None' such as "from . import foo"
+        if node.module is None:
+            self.logger.debug("Old ImportFrom: from %s import %s, %s:%s" % (node.module, [format_alias(x) for x in node.names], self.filename, node.lineno))
+            tgt_level = node.level 
+            current_module_namespace = self.module_name.rsplit('.', tgt_level)[0]
+            tgt_name = current_module_namespace
+            node.module = tgt_name
+            node.level = 0
+            self.logger.debug("New ImportFrom: from %s import %s, %s:%s" % (node.module, [format_alias(x) for x in node.names], self.filename, node.lineno))
```

Comment:
This debug message is fine, since `node.module` now points to the absolute module.

Suggest only one change, rewording to "ImportFrom (resolved):" instead of "New ImportFrom:".